packages/react-static-plugin-preact/package.json

```diff
@@ -15,12 +15,11 @@
     "preversion": "yarn build"
   },
   "peerDependencies": {
-    "preact": "^8.5.1",
-    "preact-compat": "^3.19.0"
+    "preact": "^10.4.7"
   },
   "dependencies": {
-    "preact": "^8.5.1",
-    "preact-compat": "3.19.0"
+    "@prefresh/webpack": "^0.8.2",
```

Comment:
eslint was complaining when i tried to install this as a dev dependency for the plugin and imported it in `node.api.js`. Should this be a regular dependency or dev dependency?

Reply:
IIRC, `node.api` only runs when _compiling_ a react static site. To me that's a _development_ thing, so a devdependency. Do you per chance know which rule it was? It should be turned off for `node.api` files.

Reply:
the rule is `eslint(import/no-extraneous-dependencies)`

I just looked at the eslintignore and it seems like using only one star as wildcard is not enough to reach the nested src folder in the packages:

```
# Packages build into their root
packages/*/node.api.js
packages/*/browser.api.js
```

changing this to 
```
# Packages build into their root
packages/**/node.api.js
packages/**/browser.api.js
```
seems to correctly ignores the files